### PR TITLE
ContextMenu: Add scrolling demo for .Net MVC and Core

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

**Review by iBat — DISMISSED** [review]

File: MVCDemos/Views/ContextMenu/Scrolling.cshtml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@(Html.DevExtreme().ContextMenu()
+    .Width(200)
+    .Target(".target-area")
+    .OnItemClick("contextMenu_ItemClick")
+    .DataSource(d => d.Mvc().LoadAction("GetScrollingProducts"))
+)
+
+<script>function contextMenu_ItemClick(e) {
+        if(!e.itemData.Items) {
+            DevExpress.ui.notify("The \"" + e.itemData.text + "\" item was clicked", "success", 1500);
```
Comment (ksercs) [inline MVCDemos/Views/ContextMenu/Scrolling.cshtml]: Using of string literals is more readable I think

File: MVCDemos/Views/ContextMenu/Scrolling.cshtml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        }
+    }</script>
```
Comment (ksercs) [inline MVCDemos/Views/ContextMenu/Scrolling.cshtml]: fix indents please

File: MVCDemos/Controllers/ContextMenuController.cs [hunk 3]
```diff
@@ -9,8 +10,17 @@ public class ContextMenuController : Controller {
...
+        public ActionResult Scrolling() {
+            return View();
+        }
+
         public ActionResult Templates() {
             return View();
         }
+
+        [HttpGet]
+        public ActionResult GetScrollingProducts(DataSourceLoadOptions loadOptions) {
```
Comment (ksercs) [inline MVCDemos/Controllers/ContextMenuController.cs:22]:
```suggestion
        public ActionResult GetProducts(DataSourceLoadOptions loadOptions) {
```

File: MVCDemos/Controllers/ContextMenuController.cs [hunk 4]
```diff
@@ -1,6 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿using DevExtreme.AspNet.Data;
+using DevExtreme.AspNet.Mvc;
+using DevExtreme.MVC.Demos.Models.SampleData;
+using Newtonsoft.Json;
```
Comment (ksercs) [inline MVCDemos/Controllers/ContextMenuController.cs]: Is it really necessary here? Does not the demo work without it?

File: NetCoreDemos/wwwroot/css/DemosStyles/ContextMenu/Scrolling.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    color: var(--dx-color-text);
+    background-color: var(--dx-color-border);
+    height: 300px;
+    max-width: 600px;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
```
Comment (mpreyskurantov) [inline NetCoreDemos/wwwroot/css/DemosStyles/ContextMenu/Scrolling.css]: Redundant empty line. Better remove and add ending line break instead

Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/wwwroot/css/DemosStyles/ContextMenu/Scrolling.css]: [da27f48](https://github.com/DevExpress/devextreme-demos/pull/3203/commits/da27f489909f7ec7cf3495811f9afbd46857a1c9)

File: MVCDemos/Controllers/ContextMenuController.cs (hunk 4 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Controllers/ContextMenuController.cs]: [3e130c0](https://github.com/DevExpress/devextreme-demos/pull/3203/commits/3e130c098fbe91e5583e340f3b1092a8140650ae)

File: MVCDemos/Views/ContextMenu/Scrolling.cshtml (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/ContextMenu/Scrolling.cshtml]: [da27f48](https://github.com/DevExpress/devextreme-demos/pull/3203/commits/da27f489909f7ec7cf3495811f9afbd46857a1c9)

File: MVCDemos/Views/ContextMenu/Scrolling.cshtml (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/ContextMenu/Scrolling.cshtml]: [da27f48](https://github.com/DevExpress/devextreme-demos/pull/3203/commits/da27f489909f7ec7cf3495811f9afbd46857a1c9)

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**